### PR TITLE
Feat/home grown value set filter#7

### PR DESCRIPTION
**Pull request by theisuru**
- Right-side filtering panel with facets
- Facets are defined in a configuration file similar to column_definition in ag-grid

ticket: ebi-ait/data-catalogue#7

### CODE REVIEW

File: src/app/grid/grid.component.html [hunk 1]
```diff
@@ -1,32 +1,31 @@
...
-                  [value]="facet">{{facet}}</mat-option>
-    </mat-select>
-  </mat-form-field>
-</div>
+  <div class="grid-wrapper">
+    <div class="grid-header">
+      <div class="grid-header-title">Search Datasets: </div>
+      <input type="text" id="filter-text-box" class="grid-header-input" placeholder="search by genes, cell lines, etc.." (input)="onFilterTextBoxChanged()"/>
+      <div class="grid-header-options">
+        <button class="grid-header-options-ctl" disabled>Columns</button>
```
Comment (amnonkhen) [inline src/app/grid/grid.component.html:14]: I would remove this. I don't think they asked for this initially.

File: src/app/url-cell-renderer.component.ts [hunk 2]
```diff
@@ -1,27 +1,26 @@
...
 
   refresh(params: ICellRendererParams) {
-    this.params = params;
-    // As we have updated the params we return true to let AG Grid know we have handled the refresh.
-    // So AG Grid will not recreate the cell renderer from scratch.
+    this.cellValue = this.getValueToDisplay(params);
     return true;
   }
+
+  getValueToDisplay(params: ICellRendererParams) {
```
Comment (amnonkhen) [inline src/app/url-cell-renderer.component.ts:23]: A more standard name would be "renderValue"

File: src/app/footer/footer.component.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.footer {
+  height: 30px;
+  width: 100%;
+  background: #f1f1f1;
```
Comment (amnonkhen) [inline src/app/footer/footer.component.scss:4]: I would add explicit color values in a separate css that would be project specific and reference them here.

Comment (theisuru) [reply src/app/footer/footer.component.scss:4]: Postponing this task to do when we are adjusting the UI

**Review by amnonkhen — APPROVED** [review]

**Pull request merged**